app/Extensions/AppDelegate+UserProfile.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Created by Backbase on 07/02/2024.
+//
+
+import Resolver
+import Backbase
+import UserManagerClient2
+import UserProfileJourney
+import UserManagerUserProfileUseCase
+
+extension AppDelegate {
```

Comment:
Please move this into its own handler class and remove it from the AppDelegate to properly separate concerns. It can be a class with proper methods or struct with static methods.

Reply:
Why would we need that struct? This method will only be called once during the life time of this app. That is why it is set up the same way as the rest of the use cases and clients in the app.

Reply:
The current way for the other use-cases and configurations is also not okay and we have a technical debt ticket to tackle it in the near future. Currently the AppDelegate already has too many responsibilities, which goes against the separation of concerns and of course it hinders testability.